.travis_commands.bash

```diff
@@ -19,8 +19,8 @@ ponyc-build-packages(){
 }
 
 ponyc-build-docs(){
-  echo "Installing mkdocs..."
-  sudo -H pip install mkdocs
+  echo "Installing mkdocs and offical theme..."
+  sudo -H pip install ponylang-mkdocs
```

Comment:
This command errors out. Shouldn't the package be `mkdocs-ponylang`?

Reply:
You are right. Will fix that ASAP.